src/ConfigManager.h

```diff
@@ -25,6 +25,16 @@
     using WebServer = ESP8266WebServer;
 #endif
 
+#define DEBUG_MODE false
```

Comment:
this is shitty IMO ... going to change

Reply:
Not knowing how/when this will be defined and redefining could leads to oddities 
https://stackoverflow.com/questions/42151118/how-to-override-define-in-arduino-library-header

Reply:
I see...which could be a better solution for that?

Reply:
working through it .. Started with a class method, but that felt wrong. Your way was the simplest, but could produce an oddity --- it may jus be a variable as opposed to a constant/define